RELEASING.md

```diff
@@ -0,0 +1,15 @@
+1. Set the version name. The following files need to be updated:
+    - `android/Cargo.toml`
+    - `android/examples/example-instructions.md`
+    - `android/gradle.properties`
+    - `java/Cargo.toml`
+    - `java/tests/version.sbt`
+1. Use `sbt` to release the jar file.
+    1. Make sure you have a sonatype account with write access to com.ironcorelabs. You'll need to type your username/password.
```

Comment:
You don't need this if you have a file in your global sbt configuration, at least for the sbt directions.

Reply:
Updated.